src/_pytest/terminal.py

```diff
@@ -255,6 +252,28 @@ def _determine_show_progress_info(self):
             return False
         return self.config.getini("console_output_style") in ("progress", "count")
 
+    @property
+    def verbosity(self):
+        return self.config.option.verbose
+
+    @property
+    def showheader(self):
+        return self.verbosity >= 0
+
+    @property
+    def showfspath(self):
```

Comment:
If we want to allow this to change (as it seems to be the case), then we might as well turn this into a normal attribute initialized with `self.verbosity >= 0`. 😉

Reply:
Hmm actually nevermind, this has different semantics: unless someone changes `showfspath`, it will always be derived from `config.verbosity`. 👍